fastchat/serve/inference.py

```diff
@@ -95,6 +105,44 @@ def load_model(model_path, device, num_gpus, max_gpu_memory="13GiB",
     return model, tokenizer
 
 
+def load_compress_model(model_path, device):
+
+    # partially load model
+    tokenizer = AutoTokenizer.from_pretrained(model_path, use_fast=False)
+    base_pattern = os.path.join(model_path, "pytorch_model-*.bin")
+    files = glob.glob(base_pattern)
+    
+    with init_empty_weights():
+        config = AutoConfig.from_pretrained(model_path, low_cpu_mem_usage=True, device_map='auto',torch_dtype=torch.float16)
```

Comment:
is `device_map='auto'` required?

Reply:
It seems that `device_map` is not needed in this situation, so I removed it